docs/crashlytics/ios-setup.md

```diff
@@ -16,7 +16,7 @@ add a 'New Run Script Phase':
 In the new build phase, add a new script into the text box:
 
 ```
-"${PODS_ROOT}/Fabric/run"
+"${PODS_ROOT}/FabricCrashlytics/run"
```

Comment:
I think, given what is in the `packages/crashlytics/ios_config.sh` (and https://firebase.google.com/docs/crashlytics/upgrade-sdk), this should be:

```suggestion
"${PODS_ROOT}/FirebaseCrashlytics/run"
```